features/features.go

```diff
@@ -37,763 +37,763 @@ var (
 	allFeatureGates = map[ClusterProfileName]map[configv1.FeatureSet]*FeatureGateEnabledDisabled{}
 
 	FeatureGateConsolePluginCSP = newFeatureGate("ConsolePluginContentSecurityPolicy").
-					reportProblemsToJiraComponent("Management Console").
-					contactPerson("jhadvig").
-					productScope(ocpSpecific).
-					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-					enhancementPR("https://github.com/openshift/enhancements/pull/1706").
-					mustRegister()
```

Comment:
Please don't reformat the entire file

Reply:
goland thought it was helping, pushed without the entire file change